java/src/main/java/com/google/appengine/tools/pipeline/JobSetting.java

```diff
@@ -67,18 +67,15 @@ public String toString() {
   /**
    * An abstract parent object for String settings.
    */
+  @Getter
+  @RequiredArgsConstructor
   abstract class StringValuedSetting implements JobSetting {
 
     private static final long serialVersionUID = 7756646651569386669L;
```

Comment:
```suggestion
    private static final long serialVersionUID = 1L;
```
?

Class is the same, but just for having a better uid than we have before.